src/components/NetworkSelection/NetworkSelection.tsx

```diff
@@ -0,0 +1,96 @@
+import { ALL_CHAINS, Chain } from '@pangolindex/sdk';
+import React, { useEffect, useState } from 'react';
+import { Scrollbars } from 'react-custom-scrollbars';
+import { Box, Modal, Text, ToggleButtons } from 'src/components';
+import { ButtonFrame, ChainButton, ChainsList, CloseButton, Frame, Logo } from './styled';
+import { NetworkProps } from './types';
+
+interface MetamaskError {
+  code: number;
+  message: string;
+}
+
+enum NETWORK_TYPE {
+  MAINNET = 'Mainnet',
+  TESTNET = 'Testnet',
+}
+
+const NetworkSelection: React.FC<NetworkProps> = (props) => {
+  const { open, closeModal } = props;
+  const [mainnet, setMainnet] = useState(true);
+  const [chainListHeight, setChainListHeight] = useState(48);
+  const { ethereum } = window;
```

Comment:
there is typescript error here. Please refer `src/react-app-env.d.ts` in interface to see how to fix it.

<img width="1241" alt="Screenshot 2022-05-18 at 6 11 15 PM" src="https://user-images.githubusercontent.com/6604146/169041131-7acdfd15-a28d-4718-bcb8-14e0dbbd2521.png">